packages/components/src/widgets/PlotWidget/RelativeValuesGridChart/index.tsx

```diff
@@ -63,16 +63,20 @@ const Cell: FC<{
     return <ErrorCell />;
   }
   const jsItem = itemResult.value.asJS();
-  if (!(jsItem instanceof Map)) {
+  if (!(jsItem instanceof Object)) {
```

Comment:
Many things in JS are `instanceof` Object:
```
> class X {}; (new X()) instanceof Object
true
> [] instanceof Object
true
```

OTOH, you do try/catch below, and `rvSchema.parse` is still there, so it should be fine.